.babelrc

```diff
@@ -21,8 +21,12 @@
         regenerator: true
       }
     ],
+    "transform-class-properties",
     [
-      "transform-class-properties"
+      "transform-builtin-extend",
```

Comment:
This is needed so that "XXX instanceof SubclassedError" works... Once we stop transpiling classes we can also remove this...